webdriver/tests/bidi/script/evaluate/evaluate.py

```diff
@@ -0,0 +1,146 @@
+import pytest
+
+from webdriver.bidi.modules.script import ScriptResultException
+from webdriver.bidi.modules.script import ContextTarget
```

Comment:
nit: group imports? (no strong opinion but that seems to be the convention in the folder?)
```suggestion
from webdriver.bidi.modules.script import ContextTarget, ScriptResultException
```

Reply:
done